presto-main/src/main/java/com/facebook/presto/sql/analyzer/AggregationAnalyzer.java

```diff
@@ -611,7 +616,7 @@ private void verifyNoOrderByReferencesToOutputColumns(Node node)
         getReferencesToScope(node, analysis, orderByScope.get())
                 .findFirst()
                 .ifPresent(expression -> {
-                    throw new SemanticException(REFERENCE_TO_OUTPUT_ATTRIBUTE_WITHIN_ORDER_BY_AGGREGATION, expression, "Invalid reference to output projection attribute from ORDER BY aggregation");
+                    throw new SemanticException(REFERENCE_TO_OUTPUT_ATTRIBUTE_WITHIN_ORDER_BY_AGGREGATION, expression, "Invalid reference to output projection attribute from ORDER BY expression");
```

Comment:
You changed the exc msg but not the `SemanticErrorCode` and they are out of sync now.

Maybe let the `verifyNoOrderByReferencesToOutputColumns` return `boolean` and let callers decide which error code and with what exc msg to throw?